docs/azure_arc_jumpstart/azure_arc_data/gke/gke_postgres_terraform/_index.md

```diff
@@ -116,43 +107,43 @@ By the end of this scenario, you will have a GKE cluster deployed with an Azure
 
 * Browse to <https://console.cloud.google.com/> and login with your Google Cloud account. Once logged in, [create a new project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) named "Azure Arc Demo". After creating it, be sure to copy down the project id as it is usually different then the project name.
 
-  ![GCP new project](./01.png)
+  ![GCP new project](./05.png)
 
-  ![GCP new project](./02.png)
+  ![GCP new project](./06.png)
 
-  ![GCP new project](./03.png)
+  ![GCP new project](./07.png)
 
 * Enable the Compute Engine API for the project, create a project Owner service account credentials and download the private key JSON file and copy the file to the directory where Terraform files are located. Change the JSON file name (for example *account.json*). The Terraform plan will be using the credentials stored in this file to authenticate against your GCP project.
 
-  ![Enable Compute Engine API](./04.png)
+  ![Enable Compute Engine API](./08.png)
 
-  ![Enable Compute Engine API](./05.png)
+  ![Enable Compute Engine API](./09.png)
 
-  ![Add credentials](./06.png)
+  ![Add credentials](./10.png)
 
-  ![Add credentials](./07.png)
+  ![Add credentials](./11.png)
 
-  ![Add credentials](./08.png)
+  ![Add credentials](./12.png)
 
-  ![Add credentials](./09.png)
+  ![Add credentials](./13.png)
 
-  ![Add credentials](./10.png)
+  ![Add credentials](./14.png)
 
-  ![Create private key](./11.png)
+  ![Create private key](./15.png)
```

Comment:
Use the same alt text format in other scenarios : "Screenshot showing creating a private key"